Integrations/Cymulate/Cymulate.yml

```diff
@@ -0,0 +1,67 @@
+category: Vulnerability Management
+commonfields:
+  id: Cymulate
+  version: -1
+configuration:
+- defaultvalue: 179743a665726a99fdee3bb0d10ba3fe
+  display: x-token
+  name: x-token
+  required: true
+  type: 0
+- display: Fetch incidents
+  name: isFetch
+  required: false
+  type: 8
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
+- display: Show only 'penatrated' incidents
+  hidden: false
+  name: onlyPenatrated
+  required: false
+  type: 8
+description: Multi-Vector Cyber Attack | Breach and Attack Simulation
+display: Cymulate
+name: Cymulate
+script:
+  commands:
+  - arguments:
+    - default: false
+      description: The incident unique identifier
+      isArray: false
+      name: incident_id
+      required: true
+      secret: false
+    - default: false
+      description: The Cymulate module_type
+      isArray: false
+      name: module_type
+      required: true
+      secret: false
+    deprecated: false
+    description: This commands return full cymulate's incidents information
+    execution: false
+    name: cymulate_get_incident_info
+    outputs:
+    - contextPath: Cymulate.Incident.Payload
+      description: The Cymulatepayload that generated this incident
+      type: String
+    - contextPath: Cymulate.Incident.Name
+      description: The name of the incident
+      type: String
+    - contextPath: Cymulate.Incident.Status
+      description: The attack status
```

Comment:
Status is an Enum. Add description to all the enum, which options exists for status.